packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -365,6 +367,23 @@ removing these ingress/egress rules in order to restrict access to the default s
     
 
 
+### @aws-cdk/aws-kms:aliasNameRef
```

Comment:
Is this supposed to be here?

Reply:
Good question.  I added everything in my workspace following this change.  I had a merge from aws-cdk:main mid-change that I suspect this came from.  Let me confirm that this is just duplicative from that merge.

Reply:
It doesn't look like this was something from that merge as far as I can tell.  I don't know where this came from.

Reply:
Removing this change and rebuilding returns this change